app/assets/javascripts/mobile.js

```diff
@@ -14,11 +14,6 @@ $(function () {
 
   guideTabs.prototype = {
     init : function () {
-      // if the guideTabs are floated, quit (isn't small-screen)
```

Comment:
Let's not change anything unless it impedes this tracking.

We're currently looking into the usefulness of this component within the work to merge Guides and Travel Advice into a Multipage template.

If we find users are fine with an inline list we can remove this entire component, if we decide we're going to keep this component then we'll clean this up and dedicate the time to that.